### PR TITLE
Update Microsoft.VMwareCloudSimple provider schema

### PR DESCRIPTION
**Pull request by mpoliako**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
Note that this PR introduces several breaking changes in an API version marked as stable and will require approval from ARM.

File: specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/examples/CreateVirtualMachine.json [hunk 1]
```diff
@@ -40,92 +38,88 @@
   },
   "responses": {
     "200": {
-      "body": {
```
Comment (nschonni) [inline specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/examples/CreateVirtualMachine.json]: Pretty sure the wrapping "body" is required in all these examples

**Comment by mpoliako** [comment]
> Note that this PR introduces several breaking changes in an API version marked as stable and will require approval from ARM.

I fixed code, could you please check if it still have breaking changes

**Comment by jhendrixMSFT** [comment]
Changing operation IDs is a breaking change for SDKs as it changes the client type names and their methods.
Also changing `regionId` parameter location from client to method is an SDK breaking change.
We won't prevent you from making these changes however we track breaking changes and report them in fundamentals.  Please let me know how you want to proceed.

**Comment by mpoliako** [comment]
> Changing operation IDs is a breaking change for SDKs as it changes the client type names and their methods.
> Also changing `regionId` parameter location from client to method is an SDK breaking change.
> We won't prevent you from making these changes however we track breaking changes and report them in fundamentals. Please let me know how you want to proceed.

I would like to proceed.
These changes are done on request of Microsoft Team that generates cli client for Cloudsimple API.
No one other uses clients for today

Thank you

**Review by majastrz — APPROVED** [review]
RP owners confirmed that scope of breaking changes is limited in scope and manageable. Signing off from ARM side.

**Review by jhendrixMSFT — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
Please let me know when this is safe to merge.

**Comment by mpoliako** [comment]
@jhendrixMSFT could you please merge this pull request.
As of now new version of Cloudsimple is already in production

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**